README.md

```diff
@@ -22,7 +23,7 @@ console.log(results);
 > Hint: Assuming there is a document with a name of "Bob", why does `results` not contain an author model on the second line?
 
 ```js
-// Your answer...
+good question, I believe is because we set it to the results so there is no need to be redundant.
 ```
 
```

Comment:
We need a callback function here, otherwise results will be undefined.
var results = AuthorModel.find({name: "Bob"}, (err, results) => {
  console.log(results);
})